test/e2e-tests/support/git.go

```diff
@@ -0,0 +1,34 @@
+package support
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+type GitRepoURL struct {
+	Server   string
+	OrgName  string
+	RepoName string
+}
+
+func ParseGitURL(rawURL string) (GitRepoURL, error) {
+	gitRepoURL := GitRepoURL{}
+
+	URL, err := url.Parse(rawURL)
+	if err != nil {
+		return gitRepoURL, err
+	}
+
+	// using trim here aswell because leading and trailing / causes empty strings when slicing
+	subPaths := strings.Split(strings.Trim(URL.Path, "/"), "/")
+	if len(subPaths) != 2 {
+		return gitRepoURL, fmt.Errorf("cannot parse git repo URL, expected [scheme]://[org]/[repo] got %v", rawURL)
```

Comment:
That error message looks a little suspicious to me: should it not be expecting a server/host part also? I haven't tried this out

Reply:
Agh you are right..

[scheme]://[org]/[repo] -> https://opendatahun-io/ai-edge
[scheme]://[host]/[org]/[repo] -> https://github.com/opendatahun-io/ai-edge